mgmt/rest/server.go

```diff
@@ -516,11 +516,17 @@ func respond(code int, b rbody.Body, w http.ResponseWriter) {
 }
 
 func parseNamespace(ns string) []string {
-	if strings.Index(ns, "/") == 0 {
-		ns = ns[1:]
-	}
-	if ns[len(ns)-1] == '/' {
-		ns = ns[:len(ns)-1]
+	fc := getFirstChar(ns)
+	ns = strings.Trim(ns, fc)
+	return strings.Split(ns, fc)
+}
+
+// GetFirstChar returns the first character from the input string.
+func getFirstChar(s string) string {
+	firstChar := ""
+	for _, r := range s {
+		firstChar = fmt.Sprintf("%c", r)
```

Comment:
When I need a separator different than `/`, it doesn't find the first character:
```bash
❯ curl "http://localhost:8181/v1/metrics/|intel|/etc/plop|foo"
{
  "meta": {
    "code": 404,
    "message": "No metric found below the given namespace: /|intel|/etc/plop|foo",
    "type": "error",
    "version": 1
  },
  "body": {
    "message": "No metric found below the given namespace: /|intel|/etc/plop|foo",
    "fields": {}
  }
}
```

I suspect the first character to be `/` from the URL. 2 possible solutions:
- use the second character: `curl "http://localhost:8181/v1/metrics//intel/mock/foo"`
- use the last character: `curl "http://localhost:8181/v1/metrics/intel/mock/foo/"`